### DISCUSSION TITLE
Add more filetypes to attachments group

### DISCUSSION THREAD

**Discussion by Auratelience** [notaapp/talk#327] [Feature Requests]
Hi Nota Team

I would like to be able to add audio (.m4a) and video (.mp4, .mov, etc) to the Nota sidebar, and ideally make them linkable, although if this falls outside the Markdown spec it's not necessary.

Thank you for all the great work on Nota so far, it's an incredible app.

Auratelience

**Comment by astoilkov** [comment]
Yeah. We will add this. I also want it. You are definitely not alone, there are also other issues that talk about the same thing.
